sdk/keyvault/azure-security-keyvault-keys/test/samples/sample1-hello-world/CMakeLists.txt

```diff
@@ -14,4 +14,4 @@ add_executable (
 
 create_per_service_target_build_for_sample(keyvault sample1-hello-world)
 
-target_link_libraries(sample1-hello-world PRIVATE azure-security-keyvault-keys azure-identity get-env-helper)
+target_link_libraries(sample1-hello-world PRIVATE azure-security-keyvault-keys azure-security-attestation azure-identity get-env-helper)
```

Comment:
Why would a keyvault hello world sample link `azure-security-attestation`?

Reply:
good catch , i was using it for some prototype .